docs/visualization/path/response_time.md

```diff
@@ -87,3 +87,34 @@ plot.show()
 The horizontal axis can be changed to `system time` or `index` by changing `plot.show()` to `plot.show(xaxis_type='index')`.
 
 `[worst - best] response time` in legends refers to the difference in response time between the worst case and the best case.
+
+## TimeSeries
+
+The following scripts generate timeseries graphs of response time. Horizontal axis of the graph means timeseries and vertical axis means Response time in the best case, worst case, or all case.
```

Comment:
If it is to be similar to a `stacked_bar` section, it should look like this:
```suggestion
The following scripts generate timeseries graphs of response time. Horizontal axis of the graph means timeseries and vertical axis means `Response time` in the best case, worst case, or all case.
```

Reply:
fixed in https://github.com/tier4/CARET_doc/pull/215/commits/d3b7e8e8d1732c0db934a225caab91a14db491a9.